spec/client/watches/ApiSubject.spec.ts

```diff
@@ -350,6 +360,63 @@ describe('ApiSubject', function (): void {
 			expect(cb).toHaveBeenCalledWith(event)
 		})
 
+		it('does not invoke a callback is the new data is older than the current', async () => {
```

Comment:
@garethj2 should this say `older than or equally old to the current`? 
I'm guessing since this is based on `HDict.isNewer` this would be the case.

Or as an alternative could it be worth to also test that edge case? (Assuming we don't want the callback to be invoked with the exact same data)